### PR TITLE
feat: rewrite in haskell

### LINKED ISSUE #1: Convert to Go
While typescript is cool, you could really enhance the performance of this application by writing it in Go and using wasm to compile it to webassembly

### PR DESCRIPTION
**Pull request by jaredallard**
Fixes #1

### CODE REVIEW

**Comment by j-crowe** [comment]
I approve these changes to Snak.

**Review by j-crowe — CHANGES_REQUESTED** [review]
Please reference Buca Di Beppo in the bee movie script. Thx

**Comment by jaredallard** [comment]
@stegmanh any idea when this will be merged? we need this in ASAP

**Comment by j-crowe** [comment]
Approved

File: main.go [hunk 1]
```diff
@@ -0,0 +1,4570 @@
...
+I mean the giant pulsating flower
+
+  
+made of millions of bees!
+
+  
+Pull forward. Nose down. Tail up.
+
+  
+Rotate around it.
```
Comment (stegmanh) [inline main.go]: Rotate what? This should be clearer to the audience

**Comment by stegmanh** [comment]
Please update the commit with review feedback and we can discuss further

**Comment by jaredallard** [comment]
@stegmanh fixed as per your feedback, please let's get this merged

**Comment by stegmanh** [comment]
@jaredallard Im concerned with your chose of `Go` because of it's lack of immutable variables and pure functions. 

How about we get this in Haskell and I think I can seriously consider merging it within the fortnite. Thank you

**Comment by jaredallard** [comment]
@stegmanh will get on it ASAP

**Comment by jaredallard** [comment]
@stegmanh this has been done, please get this merged ASAP. need to use immediately

**Comment by stegmanh** [comment]
It looks like you merely updated the PR name without a commit. I have to say. I respect the sheer audacity

**Comment by jaredallard** [comment]
@stegmanh Apologies, I appear to have forgotten to push my code... Will fix immediately.

**Comment by jaredallard** [comment]
@stegmanh updated, please merge asap!

**Comment by stegmanh** [comment]
@jaredallard This looks really nice. Im not seeing any tests though?

**Comment by jaredallard** [comment]
@stegmanh Of course, will get that added

**Comment by jaredallard** [comment]
@stegmanh After many difficult nights, contemplating this very important feature... I've decided, along with many other developers opinions (@j-crowe among others), that good code written by good developers does not require tests. This can be seen at the greatest of companies, who's very semblance in life is to create solutions to the world's problems.

**Comment by jaredallard** [comment]
@stegmanh please get this merged as soon as possible. I am facing a lot of external pressure to get this in.

**Comment by jaredallard** [comment]
@stegmanh if we don't get this merged by tomorrow I will literally catch on fire. You have no idea the pressure I'm under to get this in

**Comment by stegmanh** [comment]
Hey @jaredallard - I apologize for getting back to you so late, it's been beesy around these parts. 

It looks like instead of a haskell implementation you accidentally committed the entire script of the Bee movie? If this unblocks you I'd still be happy to merge..

**Pull request closed without merging**